fs/info.py

```diff
@@ -71,6 +69,15 @@ def get(self, namespace, key, default=None):
         except KeyError:
             return default
 
+    def _require_namespace(self, namespace):
```

Comment:
@althonos This is a notable change. If a required namespace is not present, it raises a `MissingInfoNamespace` which is a subclass of `AttributeError`.

The motivation was the `islink` could have 3 return values; False, True and None. And I really didn't like that.

What do you think?